src/_11ty/syntax/dash-light.js

```diff
@@ -0,0 +1,137 @@
+export default {
+  name: 'dash-light',
```

Comment:
This is the theme for the site, right? Could use just a line or two to explain it!

Reply:
This was originally a JSON file so I couldn't easily add comments. I didn't think to add any when I converted it to a JS file. Good thinking 🧠